### ISSUE TITLE
Comments from info@

### ISSUE THREAD

**Issue by rhoesing** [GSA/fedramp-tailored#49]
Hey guys,

Attached is Coalfire’s feedback for the Tailored baseline public draft. Please let us know if you have any questions.

-AW

Andrew Williams

Product Director | PUBLIC SECTOR
[Tailored Baseline PMO Feedback.docx](https://github.com/GSA/fedramp-tailored/files/861462/Tailored.Baseline.PMO.Feedback.docx)

**Comment by FedRAMP** [comment]
Thank you for your comments. For now, we are leaving those list of attested controls as-is. We are aware there is some risk in the selection of controls where attestation is allowed. We intend to re-
evaluate the success of this approach after some systems have passed through the FedRAMP Tailored process.

APPENDIX D - FedRAMP Tailored LI-SaaS Continuous Monitoring Guide is now included and contains guidance for addressing vulnerabilities in similar timeframes (30 days for high, 90 days for moderate).
At this time FedRAMP does not have estimates for the time or cost reductions associated with FedRAMP Tailored. Once a few systems have used the process, we intend to perform those estimates and publish them.

We used the NIST SP 800-53 Revision 4 baseline for Low systems as a starting point. The controls you list are not part of that baseline. There are many controls excluded by NIST for Low, and we are electing to follow that baseline unless we are presented with a compelling reason to add a non-Low requirement for Low systems. An AO always has the latitude to add controls as appropriate.

There was overwhelming feedback regarding a threshold for PII. We have defined a PII allowance for systems that that collect only enough PII to provide a login capability. While FedRAMP is asserting a system may be consider low with only this threshold of PII, each agency's AO will always have the final decision.
